content/05.main-text.md

```diff
@@ -278,41 +278,68 @@ The second uses a Cypher query, prepared by [`hetnetpy.neo4j.construct_dwpc_quer
 Both of these implementations require traversing all paths between the source and target node.
 Hence, they are computationally cumbersome despite optimizations [@doi:10.15363/thinklab.d187].
 
-An alternative approach counts paths by multiplying adjacency matrices.
-However, this approach actually counts walks,
-since it includes sequences of edges that traverse a single node (i.e trail) or edge (i.e. walk) multiple times.
+Since our methods only require weighted counts, not fully enumerated paths, adjacency matrix multiplication presents an alternative approach.
```

Comment:
Should we say `degree-weighted counts`? I want to avoid confusion with weighted edges in an adjacency matrix. We used binary adjacency matrices, but some readers might associate "weighted" with edge weights.

Here and below.